### DISCUSSION TITLE
Subscript and Superscript is rendered as normal text

### DISCUSSION THREAD

**Discussion by guategeek** [requarks/wiki#3721] [Error / Bug Report]
**Describe the bug**
Superscript and Subscript is rendered as normal text.

**To Reproduce**
Place the following text in a markdown file:
`This text is ~subscript~ and this text is ^superscript^.`

**Expected behavior**
The text should actually be formatted in the preview or on the page as subscript or superscript.

**Host Info:**
 - OS: web Node Server (Heroku)
 - Wiki.js version: 2.0.12
 - Database engine: Heroku Postgres Hobby Dev

**Comment by isiah-lloyd** [comment]
I was not able to reproduce this bug on the latest development version. Perhaps this issue is fixed?

**Comment by birdwing** [comment]
We are seeing this same bug on the latest release of Wiki.js as well.

* OS: CentOS (Docker Container)
* Wiki.js version: 2.5.170
* Database engine: PostgresSQL 11.10
